app/file.go

```diff
@@ -533,7 +534,7 @@ func (a *App) generateThumbnailImage(img image.Image, thumbnailPath string, widt
 	}
 
 	if _, err := a.WriteFile(buf, thumbnailPath); err != nil {
-		mlog.Error(fmt.Sprintf("Unable to upload thumbnail path=%v err=%v", thumbnailPath, err))
+		mlog.Error(fmt.Sprintf(utils.T("api.file.handle_images_forget.upload_thumb.error"), thumbnailPath, err))
```

Comment:
Looks like some merge conflicts were resolved the wrong way -- we're moving away from localizing error message logs.

Reply:
OK, I‘d recovert it

Reply:
model/client4.go CreateEmoji if ... else if ...NOTHING the end.
Even though I personally think that this way of writing is too strange to understand the code, it still looks like it